code/obj/machinery/computer/bank_records.dm

```diff
@@ -99,9 +99,10 @@
 
 			if ("edit_wage")
 				var/id = params["id"]
+				var/isSiliconUser = text2num_safe(params["isSiliconUser"])
```

Comment:
Do not trust the params for this. Infer from the user of the UI, not passing back as a param.